### PR TITLE
Awesome books: with ES6

### PR DESCRIPTION
**Pull request by myaserkhan**
- Created a new directory and initialize it with git.
- Used npm init -y command to create package.json file.
- Created the entry point for your JavaScript code called index.js
- Created index.css file for the styles.
- Created the main index.html file. Link your index.js and index.css files there.
- Created a directory called modules.
- Divided the code into modules. Save each module in a separate file in the modules dir. Import modules in the index.js file.
- Used a library [Luxon](https://moment.github.io/luxon/#/install?id=es6) for displaying current date and time.

### CODE REVIEW

File: modules/myclass.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  deleteBtn.innerHTML = 'Remove';
+  deleteBtn.addEventListener('click', () => {
+    library.removeBook(bookObj.id);
+  });
+  book.appendChild(deleteBtn);
+  bookList.appendChild(book);
+  if (localStorage.getItem('library').length > 1) {
+    bookList.style.border = '2px solid black';
+  }
+}
```
Comment (TedLivist) [inline modules/myclass.js]: - [ ] Kindly refactor to an arrow function

File: modules/myclass.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    this.id = id;
+  }
+}
+
+function getInput() {
+  const title = document.getElementById('title');
+  const author = document.getElementById('author');
+  const book = new Book(title.value, author.value, Math.random());
+  return book;
+}
```
Comment (TedLivist) [inline modules/myclass.js]: - [ ] With respect to the ES6 syntax requirement kindly refactor this function to an arrow function

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @MYaserKhan,

Good job so far!
✔️ Books can be added and removed
✔️ Implementation of date with luxon
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @MYaserKhan, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**